This patch was developed in-house, and submitted upstream:
http://bugs.python.org/issue26414

--- Python-3.7.1/Lib/posixpath.py	2018-11-28 07:12:59.490171900 +0000
+++ Python-3.7.1/Lib/posixpath.py	2018-11-28 07:00:39.498764614 +0000
@@ -18,7 +18,7 @@ pardir = '..'
 extsep = '.'
 sep = '/'
 pathsep = ':'
-defpath = ':/bin:/usr/bin'
+defpath = '/bin:/usr/bin'
 altsep = None
 devnull = '/dev/null'
 
